src/Container.js

```diff
@@ -378,71 +382,79 @@ class Container extends React.Component{
             title="Cycle Timer"
             decorator={"stopwatch"}
           />
-          <div className = "checkboxes1">
-            <CheckBox 
-              className="docked" 
-              title="End Park" 
-              id={this.assignUUID()} 
+          <div className="checkboxes1">
+            <CheckBox
+              className="docked"
+              title="End Park"
+              id={this.assignUUID()}
               value={false}
-              decorator = "teleopCheckbox"
-            />    
-            <CheckBox 
-              className="onstage" 
-              title="End Onstage" 
-              id={this.assignUUID()} 
+              decorator="teleopCheckbox"
+            />
+            <CheckBox
+              className="onstage"
+              title="End Onstage"
+              id={this.assignUUID()}
               value={false}
-              decorator = "onstage"
+              decorator="onstage"
             />
           </div>
           <div className="checkboxes1">
-            <CheckBox 
-              className="isFailure" 
-              title="Climb Failure" 
-              id={this.assignUUID()} 
+            <CheckBox
+              className="isFailure"
+              title="Climb Failure"
+              id={this.assignUUID()}
               value={false}
-              decorator = "dissapointmentCheckbox"
+              decorator="dissapointmentCheckbox"
             />
-            <CheckBox 
-              className="coopertition" 
-              title="Coopertition" 
-              id={this.assignUUID()} 
+            <CheckBox
+              className="coopertition"
+              title="Coopertition"
+              id={this.assignUUID()}
               value={false}
-              decorator = "dissapointmentCheckbox"
+              decorator="dissapointmentCheckbox"
             />
           </div>
 
         </div>
-       
+
         <div className="post-match-container">
           <h2 className="subtitle section-title">
             POST-MATCH
           </h2>
           <div>
+            <new className="slider-reference">‎ ‎ ‎ ‎ ‎ Terrible</new>
             <Slider
               title="Human Player"
               id={this.assignUUID()}
-              decorator = "slide"
+              decorator="slide"
             />
+            <new className="slider-reference">Incredible</new>
+            <div className="slider-nums">{this.sliderNumbers()}</div>
```

Comment:
In my opinion it would be better if we pack the slider numbers and the slider references into the slider widget class so that Container.js is a little more modular.